src/voting-metadata-api/.env.production.template

```diff
@@ -0,0 +1,8 @@
+NODE_ENV=production
```

Comment:
This template won't get used so to avoid confusion you may want to omit it.

Reply:
I've been keeping them around for testing, easier to find minimal configs than digging through the CI/CD for running locally

Reply:
Cool, makes sense. Was just checking just in case there was a misunderstanding of how it works